pkg/ddevapp/silverstripe.go

```diff
@@ -0,0 +1,67 @@
+package ddevapp
+
+import (
+	"fmt"
+	"github.com/ddev/ddev/pkg/fileutil"
+	"github.com/ddev/ddev/pkg/nodeps"
+	"github.com/ddev/ddev/pkg/util"
+	"os"
+	"path/filepath"
+)
+
+// isSilverstripeApp returns true if the app is of type Silverstripe
+func isSilverstripeApp(app *DdevApp) bool {
+	return fileutil.FileExists(filepath.Join(app.AppRoot, "vendor/bin/sake"))
```

Comment:
```suggestion
	return fileutil.FileExists(filepath.Join(app.AppRoot, app.ComposerRoot, "vendor/bin/sake"))
```
Problem with this detection is that if somebody changes the vendor or the bin dir in the `composer.json` this will not work. Please add a TODO comment so we are able to fix it with #5058 where a composer.json parser is introduced to fix issues like this.

Reply:
I'm wondering how realistic is it that someone would do such a thing?

Not that it shouldn't be included, just curious, I never heard anyone actually do that ;)

Reply:
I've added the todo, by the way :)

I'm interested to see how it's going to be resolved